### ISSUE TITLE
How to control Language select in Transcribe task

### ISSUE THREAD

**Issue by Alonnasi** [argmaxinc/WhisperKit#235]
Hey there 🙏
I'm tryin to use WhisperKit package with giving it a audio file which i need it transcribed with Word and Timings for each word.

As i'm testing the capabilities, I have couple of questions to know if this Kit will be good for me.
Any help will be welcomed 😇

1. How can i set a transcribe task with specific Language Spoken to detect? Do I need specific Model for certain languages?
2. How can i get Words Timings? Do I need specific Model?
3. How can i download and load specific Models? (in order to cut down the loading time for the Kit init)

Thank you guys again - and Happy coding!

**Comment by ZachNagengast** [comment]
1. The library detects the language `DecodingOptions` if `detectLanguage: true` is set. You can also specify the source language by setting `DecodingOptions(language: "fr")` to the 2 letter language code, in this example french.
2. Word timings are returned in the `TranscriptionResults` object if the model has alignment heads, most models have them. Checkout our example app for how to use this [WhisperAX](https://github.com/argmaxinc/WhisperKit/tree/main/Examples/WhisperAX). Our unit tests also cover most cases https://github.com/argmaxinc/WhisperKit/blob/main/Tests/WhisperKitTests/UnitTests.swift
3. You can either manage the model downloading yourself using the `modelFolder` init property, or simply pass the model name into the init with parameter `download: true` to download the associated model name from this repo: https://huggingface.co/argmaxinc/whisperkit-coreml

Hope that helps!

**Comment by Alonnasi** [comment]
Thank you so much for the quick response and all the info! Very much helpful!

I did managed to set the DecodingOptions correctly, and the transcribe task work very well!

The only thing left is to get the Word Timings.

I've noticed a property called wordTimestamps in the DecodingOptions, when setting it to True -> Word Timings are calculated, with no need of setting specific Model whatsoever.

Thank you again, and good luck! 🍀
